### PR TITLE
Bug questionnaire

### PR DESCRIPTION
**Pull request by kimc0de**
What I fixed: 
**Recruiter:** 
- Fix the bug in the last tab of benefit, now when the user checks the checkboxes, they dont need to write in input field
- Replace underscore with space when the values are shown in the profile or edit page
- Change salary question (we just ask desired salary, no specified on monthly or annually)
 
**Candidate**
- Remove remote in job type dropdown
- Change wording of the aboutme question
- Delete unused file: javascript/questionnaire/candidate_question.js

**Both**
- move the script from layout.ejs to the end of the new.ejs files
- add css input border

### CODE REVIEW

File: views/jobs/new.ejs [hunk 1]
```diff
@@ -126,7 +126,7 @@
           <!-- 9. Do you offer any of these options? -->
           <div class="tab">
             <section class="questions">
-              <div class="sentence">What benefits or work-related extras do you offer?</div>
+              <div class="sentence">What about the benefits you offer and work culture of the company?</div>
```
Comment (lyrated) [inline views/jobs/new.ejs:129]: I wouldn't put the "What about the..." like this. I don't have a good alternative for now so maybe **let's discuss this on Wednesday with everyone**... also maybe showing some other options (right now we only show benefits, we could also show something like "regular feedback" and so on)

**Review by lyrated — CHANGES_REQUESTED** [review]

File: views/thanks.ejs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 <div class="jumbotron">
       <h1 class="display-4">AWESOME !</h1>
-      <p class="lead">You've successfully singed up to REM, look forwards to new matches!</p>
+      <p class="lead">You've successfully singed up to ReMatch, look forwards to new matches!</p>
```
Comment (lyrated) [inline views/thanks.ejs:4]: here, it says "look forwards" -> it should say "forward" without s :D

**Review by joshuameiser — APPROVED** [review]
Everything seems alright. I don't really remember having underscores on the showSingleJob pages though, so I can't really comment on that specific change.

**Comment by lyrated** [comment]
Heyy, I remembered when you said you had this problem when you clicked on the input, it didn't have this border? In the end I had this too (not on the questionnaire, but it was only on edit pages). In bootstrap-tagsinput.css is a css rule for `.bootstrap-tagsinput input:focus` - just delete this then it should be fixed^^
I know you have to prepare for exams now so if you can't fix these things + merge until Wednesday or Friday, let us know so we can assign someone to do it~

File: views/jobs/new.ejs (hunk 1 above)
Comment (kimc0de) [reply views/jobs/new.ejs:129]: yup let me know after all bc i wont be able to join on wednesday <3

**Pull request merged**